src/main/java/org/example/api/DeliveryEmployeeService.java

```diff
@@ -29,4 +29,14 @@ public int createDeliveryEmployee(DeliveryEmployeeRequest employee) throws Proje
             return id;
 
     }
+
+    public DeliveryEmployee getDeliveryEmployeeById (int id) throws SQLException, ProjectException {
+            DeliveryEmployee deliveryEmployee = deliveryEmployeeDao.getDeliveryEmployeeById(id);
+
+                if(deliveryEmployee == null) {
+                    throw new ProjectException();
```

Comment:
Is this the right exception to throw here?

Reply:
Changed to SQLException